src/Symfony/Component/Console/Command/Command.php

```diff
@@ -164,6 +164,9 @@ public function isEnabled(): bool
      */
     protected function configure()
     {
+        if (!$this->code && \is_callable($this)) {
+            $this->code = new InvokableCommand($this, $this(...));
```

Comment:
This downside of this is we're now creating a self-referencing class. Might not be an issue in practice since we won't create several instances of the command object, but still worth to have in mind and prevent writing such code in the generic case.

Reply:
Completely agree here! I'm wondering if there is an alternative solution for this case…